client/src/App.js

```diff
@@ -59,6 +59,13 @@ function App() {
                     success
                     // isPrivate
                   />
+                  <Route
+                    exact
+                    path={navRoutes.ADMIN.EDIT_STEP}
```

Comment:
This is attributed to the wrong role. Only super admin can edit the steps

Reply:
done